doc-cheat-sheet.rst

```diff
@@ -0,0 +1,611 @@
+:orphan:
+
+=========================
+Documentation cheat sheet
+=========================
+
+The documentation files use `reStructuredText (rST) <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_ syntax. |x wokeignore:rule=master x|
+
+See the following sections for syntax help and conventions.
+
+For general style conventions, see the `Canonical Documentation Style Guide <https://docs.ubuntu.com/styleguide/en>`_.
+
+Headings
+========
+
+.. list-table::
+   :header-rows: 1
+
+   * - Input
+     - Description
+   * - .. code::
+
+          Title
+          =====
```

Comment:
Should we use overlines or not? I have no strong opinion on that, but I think @evildmp didn't like them?

Reply:
The Charmed OpenStack guide is itself aligned with the upstream [OpenStack Documentation Contributor Guide](https://docs.openstack.org/doc-contrib-guide/rst-conv/titles.html#cg-titles).

Reply:
Updated.

Reply:
I don't use overlines because I think they look untidy, but otherwise follow the [reStructuredText guidelines](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) with my team. We also put our links at the bottom of each page for ease of access. I would tend to advocate the use of rST guidelines in an rST-written doc, since their docs/guidelines will be the first to update whenever rST does.

Reply:
I don't see that document as a style guide though - it states how you can do things, but it doesn't give any recommendations or conventions.